user_sync/config.py

```diff
@@ -323,6 +323,11 @@ def get_directory_connector_options(self, connector_name):
         """
         options = {}
         connectors_config = self.get_directory_connector_configs()
+
+        if ( 'okta' in connectors_config.value) and (connector_name == 'ldap'):
+            if connectors_config.accessed_keys is not None:
+                raise AssertionException("Failed to match request : for Okta Connector type receiving LDAP as input type from commnand line")
```

Comment:
If you make the check more generic you'll need to make this message generic as well.  I'd do something like this:

```python
raise AssertionException("Config file must be specified for connector type '{}'".format(connector_name))
```

Reply:
modified. please review